ScreensPanel.c

```diff
@@ -196,91 +195,76 @@ static HandlerResult ScreensPanel_eventHandlerNormal(Panel* super, int ch) {
       case KEY_NPAGE:
       case KEY_PPAGE:
       case KEY_HOME:
-      case KEY_END: {
+      case KEY_END:
          Panel_onKey(super, ch);
          break;
-      }
       case KEY_F(2):
       case KEY_CTRL('R'):
-      {
          startRenaming(super);
          result = HANDLED;
          break;
-      }
       case KEY_F(5):
       case KEY_CTRL('N'):
-      {
          if (this->settings->dynamicScreens)
             break;
          addNewScreen(super);
          startRenaming(super);
          shouldRebuildArray = true;
          result = HANDLED;
          break;
-      }
       case KEY_UP:
-      {
          if (!this->moving) {
             Panel_onKey(super, ch);
             break;
          }
-         /* else fallthrough */
-      } /* FALLTHRU */
+         /* FALLTHRU */
       case KEY_F(7):
       case '[':
       case '-':
-      {
          Panel_moveSelectedUp(super);
          shouldRebuildArray = true;
          result = HANDLED;
          break;
-      }
       case KEY_DOWN:
-      {
          if (!this->moving) {
             Panel_onKey(super, ch);
             break;
          }
-         /* else fallthrough */
-      } /* FALLTHRU */
+         /* FALLTHRU */
       case KEY_F(8):
       case ']':
       case '+':
-      {
          Panel_moveSelectedDown(super);
          shouldRebuildArray = true;
          result = HANDLED;
          break;
-      }
-      case KEY_F(9):
-      //case KEY_DC:
-      {
+      case KEY_F(9): //case KEY_DC:
```

Comment:
Should investigate why the `case KEY_DC:` line is commented out here.
That comment on the same line as `case KEY_F(9):` is surely wrong.

Reply:
FYI: This was introduced by commit 72ba20fa5 (committed by @natoscott) already in the original commit. Didn't investigate further in the upstream code on Hisham's old branch where this has been adapted from.